### PR TITLE
restore ability to provide a comma-separated list of Cassandra hosts

### LINKED ISSUE #631: Cassandra destination cmd line switch does not allow multiple hosts
Previously `-zipkin.store.cassie.dest`command line switch used to allow a list of host:port pairs. The new switch `-zipkin.store.cassandra.dest` only allows one.

### PR DESCRIPTION
**Pull request by yurishkuro**
closes  #631

### CODE REVIEW

File: zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactory.scala [hunk 1]
```diff
@@ -58,9 +58,11 @@ trait CassandraSpanStoreFactory {self: App =>
   }
 
   def addContactPoint(builder: Cluster.Builder): Cluster.Builder = {
-    val contactPoint = HostAndPort.fromString(cassandraDest())
+    val contactPoints = cassandraDest().split(",").map(HostAndPort.fromString)
```
Comment (codefromthecrypt) [inline zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactory.scala:60]: this is a little complex.. why not just split on comma, then pass that directly to `.addContactPoints(contactPoints:_*)`

Comment (codefromthecrypt) [inline zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactory.scala:60]: guessing this is to support port overrides... nm

**Comment by codefromthecrypt** [comment]
mind updating the spec? When done, I'll help merge this.
https://github.com/openzipkin/zipkin/blob/master/zipkin-cassandra/src/test/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactorySpec.scala

**Comment by codefromthecrypt** [comment]
ps in the future, you can just open a PR when you are proposing both the problem and the fix.

**Comment by yurishkuro** [comment]
@adriancole added test

File: zipkin-cassandra/src/test/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactorySpec.scala [hunk 2]
```diff
@@ -36,4 +36,14 @@ class CassandraSpanStoreFactorySpec extends FunSuite with Matchers {
...
     )
   }
+
+  test("zipkin.store.cassandra.dest as host1:port,host2 (default port)") {
+    TestFactory.nonExitingMain(Array(
+      "-zipkin.store.cassandra.dest", "1.1.1.1:9143,2.2.2.2"
+    ))
+
+    TestFactory.addContactPoint(Cluster.builder()).getContactPoints should be(
+      asList(new InetSocketAddress("1.1.1.1", 9143), new InetSocketAddress("2.2.2.2", 9042))
```
Comment (codefromthecrypt) [inline zipkin-cassandra/src/test/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactorySpec.scala:46]: beautiful!

**Comment by codefromthecrypt** [comment]
great job. Can you squash these into a single commit? I'll merge on green after.

**Comment by yurishkuro** [comment]
@adriancole squashed to 1

**Comment by michaelsembwever** [comment]
thanks @yurishkuro !

**Pull request merged**